tests/aws/services/sns/test_sns.py

```diff
@@ -3649,6 +3649,186 @@ def _get_messages(amount: int) -> list[dict]:
         assert messages[0]["Source"] == sender_address
 
 
+class TestSNSPlatformApplicationCrud:
+    @markers.aws.manual_setup_required
+    def test_create_platform_application(self, aws_client, snapshot, cleanups):
+        name = f"platform-application-{short_uid()}"
+        platform = "ADM"
+        # these values need to be extracted from a real amazon developer account if tested against AWS
+        # https://developer.amazon.com/settings/console/securityprofile/overview.html
+        client_id = "dummy"
+        client_secret = "dummy"
+        attributes = {"PlatformPrincipal": client_id, "PlatformCredential": client_secret}
+        response = aws_client.sns.create_platform_application(
```

Comment:
question: any reason we're not using `sns_create_platform_application` here instead of the manual cleanup?

Reply:
The reason was that I apparently misread the fixture/mixed something up, and thought it won't return the whole response. And I wanted to snapshot the whole response for the create call at least once. Thanks for pointing it out!